src/main/java/marquez/api/GetAllNamespacesResponse.java

```diff
@@ -0,0 +1,25 @@
+package marquez.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.List;
+import marquez.core.models.Namespace;
+
+public class GetAllNamespacesResponse {
+
+  List<GetNamespaceResponse> namespaces = new ArrayList<>();
+
+  public GetAllNamespacesResponse(@JsonProperty("namespaces") List<Namespace> namespaceList) {
+    for (Namespace n : namespaceList) {
+      // TODO: Adjust this once we have a mapper in place
```

Comment:
You wouldn't need a mapper and can avoid transformations by:
```java
this.namespaces = namespaces
```

Reply:
They're different types right? Are you suggesting a more elegant way transform from one to another, or are you suggesting changing the input type?